### PR TITLE
Add tic tac toe implementation

### PR DESCRIPTION
**Pull request by tanviparmar1989**
Changes include:
- Implementation of Tic Tac Toe game.

### CODE REVIEW

File: src/main/java/TicTacToeCLI.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        Scanner in = new Scanner(System.in);
+
+        //initialize to class level.
+        //move the array from main.
+        // iterate
+        char[][] ticTacToeArray = {{'1', '2', '3'}, {'4', '5', '6'}, {'7', '8', '9'}};
+        ticTacToe.displayBoard(ticTacToeArray);
+
+
+//        System.out.println("Player1 - Please enter your name");
```
Comment (prashantc8809) [inline src/main/java/TicTacToeCLI.java]: Please don't commit commented lines of code to git.

**Review by prashantc8809 — CHANGES_REQUESTED** [review]
Please review my comments.

File: src/main/java/TicTacToeCLI.java (hunk 1 above)
Comment (tanviparmar1989) [reply src/main/java/TicTacToeCLI.java]: Done.

**Comment by tanviparmar1989** [comment]
Implemented TicTacToe

**Pull request merged**